app/controllers/application_controller.rb

```diff
@@ -8,10 +8,11 @@ class ApplicationController < ActionController::Base
   before_action :update_hit_counter, only: [:index, :show, :edit, :new]
   before_action :authorize
   before_action :attach_ip_in_header
+  around_action :setup_locale
```

Comment:
why around_action not before_action?

Reply:
discuss
https://guides.rubyonrails.org/i18n.html#managing-the-locale-across-requests